### PR TITLE
ROX-21025: Update recommended minimum permissions

### PR DESCRIPTION
**Pull request by ivan-degtiarenko**
Version(s):
* `rhacs-docs-4.4`

Issue:
https://issues.redhat.com/browse/ROX-21025

[Link to docs preview:
](https://71508--docspreview.netlify.app/openshift-acs/latest/operating/manage-user-access/manage-role-based-access-control-3630)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/create-a-custom-role.adoc
Comment (kcarmichael08) [inline modules/create-a-custom-role.adoc:1]: It looks like the correct file that is in use now, that is published, is actually modules/create-a-custom-role-3630.adoc - that is the one that shows up in the current online docs (https://docs.openshift.com/acs/4.3/operating/manage-user-access/manage-role-based-access-control-3630.html). 

Do you want to make the changes to the correct file instead and delete this one? It looks like for some reason, the old file (create-a-custom-role.adoc) - the one I'm commenting on now - was updated with the bulleted list of resources several years ago, but the correct file (create-a-custom-role-3630.adoc) was never updated.

Maybe at one point, both modules were needed, but it looks like right now the only module being shown in the docs is the other one, not this one.

**Review by kcarmichael08 — COMMENTED** [review]
Left one comment, otherwise LGTM. Before we can merge any changes, we have to squash commits into one commit (required by the OpenShift docs repo). (They specifically require that we don't use "squash and merge", though.) There are instructions here: https://docs.google.com/document/u/2/d/e/2PACX-1vRLKWEMHQ3DZroxZKfTu3XcrSdREr6D3oSSayBanEprXhkA2Ciyr2SQuDTYI4aIKUiOPPIQMHgjHeh8/pub#h.epes4xf06pzk

File: modules/create-a-custom-role.adoc
Comment (ivan-degtiarenko) [reply modules/create-a-custom-role.adoc:1]: I realised that this page is probably a page that was relevant before 3.63 (when we implemented scoped access control, permission sets and all that jazz). `create-a-custom-role-3630.adoc` references creation of permission set, and `create-a-custom-permission-set.adoc` contains recommended permissions list and is updated within the PR.

I think I'll just rollback changes in this file.

**Review by kcarmichael08 — APPROVED** [review]
LGTM!

**Pull request merged**